cluster/gce/gci/configure-helper.sh

```diff
@@ -2904,9 +2907,6 @@ function main() {
   KUBE_CONTROLLER_MANAGER_TOKEN="$(secure_random 32)"
   KUBE_SCHEDULER_TOKEN="$(secure_random 32)"
   KUBE_CLUSTER_AUTOSCALER_TOKEN="$(secure_random 32)"
-  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
-    KUBE_BOOTSTRAP_TOKEN="$(secure_random 32)"
```

Comment:
It's nice that we do all our global token generation in one spot, but I see the counterargument that this creates spooky action at a distance. Since KUBE_BOOTSTRAP_TOKEN has to escape from ensure-kubectl-auth, I would probably just leave it here.

Reply:
To be fair, configure-helper is almost 100% spooky action (liberal use of global state, sometimes documented in function comments). I put it in a function to be self-documenting, but I'm OK with using actual comments. Done